sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/v2/ParquetFieldIdShims.scala

```diff
@@ -27,4 +31,18 @@ object ParquetFieldIdShims {
       SQLConf.PARQUET_FIELD_ID_WRITE_ENABLED.key,
       sqlConf.parquetFieldIdWriteEnabled.toString)
   }
+
+  def tagGpuSupportWriteForFieldId(meta: RapidsMeta[_, _, _], schema: StructType): Unit = {
+    if (ParquetUtils.hasFieldIds(schema)) {
```

Comment:
This check should not be performed unless `SQLConf.PARQUET_FIELD_ID_WRITE_ENABLED` is set to true.

Reply:
done